master/internal/api_project.go

```diff
@@ -595,11 +595,11 @@ func (a *apiServer) PatchProject(
 		return nil, err
 	}
 	if currProject.Archived {
-		return nil, errors.Errorf("project (%d) is archived and cannot have attributes updated.",
+		return nil, errors.Errorf("project (%d) is archived and cannot have attributes updated",
 			currProject.Id)
 	}
 	if currProject.Immutable {
-		return nil, errors.Errorf("project (%v) is immutable and cannot have attributes updated.",
```

Comment:
curious was this trailing `.` a lint complaint or just for consistency?

Reply:
It was something complained about by one of the new linters.